src/container/styles.scss

```diff
@@ -61,11 +64,68 @@
   }
 }
 
+.with-side-media {
+  display: flex;
+  flex-direction: row;
+}
+
+.with-top-media {
+  display: flex;
+  flex-direction: column;
+}
+
+.content-wrapper {
+  flex: 1;
+  &-fit-height {
+    overflow: auto;
+  }
+  &-with-media {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.media {
+  overflow: hidden;
+
+  // stylelint-disable-next-line @cloudscape-design/no-implicit-descendant, selector-max-type
```

Comment:
Shouldn't we be using a direct descending selector (`>`) here?

Side note: Should we add some additional restrictions to the API docs or the Dev guidelines? Things like: "we only support `img`, `video`, `picture`", or "the media tag needs to be the first and only child of the `media` slot"

Reply:
I've thought about this. My thought was that there are some custom media components (like those offered by some frameworks, i.e.: `GatsbyImage` or `next/image`) that could potentially include more wrapping elements.
What do you think?

Regarding documentation, I will update! 👍

Reply:
mhm yeah, fair enough. this might lead to interesting questions down the line in terms of what we support. in case some people have weird mixed media components.

But for now I think this is fine.